hw/ip/sram_ctrl/doc/_index.md

```diff
@@ -9,12 +9,15 @@ This document specifies the functionality of the SRAM memory controller.
 The SRAM controller is a module that is a peripheral on the chip interconnect bus, and thus follows the [Comportability Specification]({{< relref "doc/rm/comportability_specification" >}}).
 
 
-The SRAM controller interfaces with the SRAM data and address scrambling device and provides CSRs for requesting the scrambling keys and reporting any parity errors that have occurred in the macro.
+The SRAM controller contains the SRAM data and address scrambling device and provides CSRs for requesting the scrambling keys and triggering the hardware initialization feature.
 
 ## Features
 
+- Lightweight scrambling mechanism based on the PRINCE prince cipher.
```

Comment:
does it make sense to just direct to the `prim_ram_1p_scr` doc for this feature? Because really `sram_ctrl` just provides the keys, and how things are scrambled is something handled a layer below right?

Reply:
yeah sounds good. That makes sense too